packages/compass-collection/src/stores/collection-tab.ts

```diff
@@ -130,5 +143,13 @@ export function activatePlugin(
   return {
     store,
     deactivate: cleanup,
+    actions: {
+      openMockDataGeneratorModal: () =>
+        store.dispatch(openMockDataGeneratorModal()),
+      closeMockDataGeneratorModal: () =>
+        store.dispatch(closeMockDataGeneratorModal()),
+      setMockDataGeneratorStep: (step: MockDataGeneratorStep) =>
+        store.dispatch(setMockDataGeneratorStep(step)),
+    },
```

Comment:
Can you explain what are you doing here? We're passing actions like that only in very old pieces of code that use reflux, not redux